subprojects/core/src/integTest/groovy/org/gradle/initialization/CalculateTaskGraphBuildOperationIntegrationTest.groovy

```diff
@@ -123,10 +144,171 @@ class CalculateTaskGraphBuildOperationIntegrationTest extends AbstractIntegratio
         taskGraphCalculations[0].result.requestedTaskPaths == [":build"]
         taskGraphCalculations[1].details.buildPath == ":"
         taskGraphCalculations[1].result.requestedTaskPaths == [":build"]
-        taskGraphCalculations[2].details.buildPath== ":b"
+        taskGraphCalculations[2].details.buildPath == ":b"
         taskGraphCalculations[2].result.requestedTaskPaths == [":compileJava", ":jar"]
     }
 
+    @ToBeFixedForInstantExecution
+    def "exposes task plan details"() {
+        file("included-build").mkdir()
+        file("included-build/settings.gradle")
+        file("included-build/build.gradle") << """
+        apply plugin:'java-library'
+        group = 'org.acme'
+        version = '1.0'
+        """
+
+        file('src/main/java/org/acme/Library.java') << """
+        package org.acme;
+
+        class Library {
+        }
+        """
+        settingsFile << """
+            includeBuild 'included-build'
+        """
+
+        buildFile << """
+                apply plugin:'java-library'
```

Comment:
Indentation seems off here.

Reply:
applied common indentation. shouldn't codenarc be able to catch those things?